### PR TITLE
Updated angular to 1.5.0-rc.0 on 06-ui-router-state-navigation

### PR DESCRIPTION
**Pull request by Creeland**
@ijones16

### CODE REVIEW

**Comment by Creeland** [comment]
Ready to merge @ijones16

File: src/app/eggly-app.js [hunk 1]
```diff
@@ -51,25 +53,25 @@ angular.module('Eggly', [
...
+        main.setEditedBookmark = setEditedBookmark;
+        main.isSelectedBookmark = isSelectedBookmark;
 
         function resetCreateForm() {
-            $scope.newBookmark = {
+            main.newBookmark = {
                 title: '',
                 url: '',
-                category: $scope.currentCategory
+                category: main.currentCategory
```
Comment (ijones16) [inline src/app/eggly-app.js]: this needs to be main.currentCategory.name

File: src/app/categories/bookmarks/bookmarks.js [hunk 2]
```diff
@@ -17,7 +17,8 @@ angular.module('categories.bookmarks', [
             })
         ;
     })
-    .controller('BookmarksCtrl', function ($scope,$stateParams  ) {
-        $scope.currentCategoryName = $stateParams.category;
+    .controller('BookmarksCtrl', function ($stateParams) {
```
Comment (ijones16) [inline src/app/categories/bookmarks/bookmarks.js]: This variable should reflect the name of the controller so main should be `var bookmark = this;`
Line 14 should say BookmarkCtrl as bookmark

**Comment by Creeland** [comment]
@ijones16 I made the requested changes

**Pull request merged**